packages/web-components/fast-foundation/src/slider/slider.template.ts

```diff
@@ -17,8 +17,8 @@ export const SliderTemplate = html<Slider>`
         aria-valuenow="${x => x.value}"
         aria-valuemin="${x => x.min}"
         aria-valuemax="${x => x.max}"
-        ?aria-disabled="${x => x.disabled}"
-        ?aria-readonly="${x => x.readOnly}"
+        aria-disabled="${x => (x.disabled ? true : void 0)}"
+        aria-readonly="${x => (x.readOnly ? true : void 0)}"
```

Comment:
```suggestion
        aria-disabled="${x => !!x.disabled}"
        aria-readonly="${x => !!x.readOnly}"
```

Reply:
We use this pattern elsewhere, but I like this one. I'll write up and issue and we can change this across the repo once the test suite is in for safety. Thanks!